CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 This file documents all notable changes to the VS Code Lizard extension.
 
+## [1.0.2] — 2022-09-29
```

Comment:
```suggestion
## Unreleased
```

I'll update this with the correct version and release date when I cut a new release. I worry users might get confused if the change log says 1.0.2 but there isn't a v1.0.2 tag.